example_docs/docs/conf.py

```diff
@@ -71,6 +70,18 @@
     'expandable_sidebar': True
 }
 
+translations_list = [
+    ('en', 'English'),
+    ('bn_BN', 'Bengali'),
+    ('fr_FR', 'French'),
+    ('de_DE', 'German'),
+    ('ja_JP', 'Japanese'),
+    ('ko_KR', 'Korean'),
+    ('pt_UN', 'Portuguese'),
+    ('es_UN', 'Spanish'),
+    ('ta_IN', 'Tamil'),
+]
```

Comment:
🎉 This looks great. Let's put just English in the example?

```suggestion
translations_list = [
    ('en', 'English'),  # Default, more languages will be added as they are ready to be published
]
```

Reply:
How come? It's useful to have multiple in the example docs. 

This `conf.py` isn't meant to be copied and pasted into new projects. It already has some things that won't make sense in other projects like this:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/086f163d336de7c827983e62f962a51e5a37b4cb/example_docs/docs/conf.py#L49-L61

Reply:
side note, if users just want english do they need the translations list at all?

Reply:
> Even if we don't want that to be the case I think we should still assume that some will. I think we can keep a longer list of languages, but I like Soolu's addition of a comment explaining that english is the default.

Fair. I will add a comment.

> side note, if users just want english do they need the translations list at all?

No. They should only set this option if they have 1+ translation. It's confusing to show the translations list and have English be the only option.